app/helpers/codelist_helper.rb

```diff
@@ -23,6 +23,11 @@ def region_select_options
     objects.unshift(OpenStruct.new(name: "Developing countries, unspecified", code: "998")).uniq
   end
 
+  def country_select_options
+    objects = yaml_to_objects(entity: "activity", type: "recipient_country", with_empty_item: false)
+    objects.unshift(OpenStruct.new(name: "Select a recipient country", code: "")).uniq
```

Comment:
Can we have this string internationalised, please 🙏

Reply:
Yep, wasn't sure where it should go but settled on this.